web-demo/interactive-console/src/main/java/ai/djl/examples/jshell/ShellController.java

```diff
@@ -19,34 +20,47 @@
 @RestController
 public class ShellController {
 
+    private Map<String, ExpireShellTask> shells;
+    private Timer timer;
```

Comment:
1. Create a static ScheduledExcutor, and run a scan every 1 - 2 minutes, the remove the from the map
2. Add a timestamp to each JShell, and update the teimstamp whenever it retrieved, the timeout will be extends if user are using it.